.github/workflows/main.yml

```diff
@@ -17,6 +17,7 @@ jobs:
           key: ${{ runner.os }}-npm-${{ hashFiles('**/package-lock.json') }}
 
       - run: npm ci --ignore-scripts
+      - run: npx run commitlint --from HEAD~${{ github.event.pull_request.commits }} --to HEAD
```

Comment:
this runs now on CI for almost 1 hour @subhod-i

Reply:
Fixed it.